src/podcast_processor/transcribe.py

```diff
@@ -18,6 +18,21 @@ class Segment(BaseModel):
     end: float
     text: str
 
+    def to_dict(self) -> dict[str, Any]:
```

Comment:
I think you can get this method for free from pydantic with `.dict()` and then `Segment(**some_dict)`

Reply:
slick! I'll give that a go